test/models/location.js

```diff
@@ -10,7 +10,6 @@ const data = require('../data')
 lab.experiment('Outlook model test', () => {
   let sandbox
 
-  // Use a Sinon sandbox to manage spies, stubs and mocks for each test.
```

Comment:
The comments relating to using a sandbox have been removed but we don't need to create a sandbox since you get one by default:

> Since sinon@5.0.0, the sinon object is a default sandbox. Unless you have a very advanced setup or need a special configuration, you probably want to only use that one.

From: https://sinonjs.org/releases/latest/sandbox/